demo/card/x-card.html

```diff
@@ -45,49 +45,51 @@
 
     properties: {
       animationConfig: {
-        value: function() {
-          return {
-            'entry': [{
-              name: 'ripple-animation',
-              id: 'ripple',
-              toPage: this
-            }, {
-              name: 'fade-out-animation',
-              node: this.$.placeholder,
-              timing: {
-                delay: 250
-              }
-            }, {
-              name: 'fade-in-animation',
-              node: this.$.container,
-              timing: {
-                delay: 50
-              }
-            }],
-
-            'exit': [{
-              name: 'fade-out-animation',
-              node: this.$.container,
-              timing: {
-                duration: 0
-              }
-            }, {
-              name: 'reverse-ripple-animation',
-              id: 'reverse-ripple',
-              fromPage: this
-            }]
-          };
-        }
+        type: Object
       },
 
       sharedElements: {
-        value: function() {
-          return {
-            'ripple': this.$.placeholder,
-            'reverse-ripple': this.$.placeholder
-          };
-        }
+        type: Object
       }
+    },
+
+    attached: function() {
+      this.animationConfig = {
```

Comment:
this is different from 1.x, as at every `attached` you'd override `this.animationConfig` and `this.sharedElements`.
WDYT of just having a check before this line like `if (this.animationConfig) return;`?